packages/web/src/pages/artist-dashboard-page/ArtistDashboardPage.module.css

```diff
@@ -159,3 +159,74 @@
   justify-content: stretch;
   margin: 12px 0px 36px;
 }
+
+.usdcContainer {
+  width: 100%;
+  flex-direction: column;
+  border-radius: var(--unit-3);
+  background: var(--static-white);
+  overflow: hidden;
+}
+
+.backgroundBlueGradient {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  gap: var(--unit-4);
+  padding: var(--unit-6);
+  background: linear-gradient(
+      315deg,
+      rgba(0, 0, 0, 0.1) 0%,
+      rgba(255, 255, 255, 0.1) 100%
+    ),
+    #2775ca;
```

Comment:
Color var? Also is this static (just double checking, don't have figma open)?

Reply:
believe it's static, copied css from figma. and don't think the color is in our system, seems like a one-off for now.